.github/workflows/release.yml

```diff
@@ -0,0 +1,66 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+      - next
+      - alpha
+      - beta
+
+concurrency:
+  group: publish-${{ github.ref }}
+
+env:
+  NODE_VERSION: 20
+  NPM_TOKEN: ${{ secrets.NPM_READ_TOKEN }}
+
+jobs:
+  verify:
+    name: Verify
+    uses: ./.github/workflows/ci.yml
+
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    needs: [verify]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build
+
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v3
```

Comment:
If we use semantic here it will only work if swr-openapi base repo uses semantic commits, but it will look at the previous release history of this library (not the base of the fork). We will probably want to make this do a release based on the version in the package file since it seems swr-openapi uses version within package file: https://github.com/htunnicliff/swr-openapi/blob/main/package.json#L4 and manual releases

That way versions will match

If we however are fine running our own version pattern (deviating from swr-openapi), then we can go forward with this